packages/components/src/components/select-wrapper/select-wrapper/select-wrapper-styles.spec.ts

```diff
@@ -16,7 +16,10 @@ describe('getComponentCss()', () => {
     [false, true, true, 'error', 'dark'],
     [true, false, true, 'error', 'dark'],
     [false, true, { base: true, xs: false, s: true, m: false, l: true, xl: false }, 'none', 'light'],
-  ])('should return correct css for disabled: %s, native: %s, hideLabel: %o, state: %s and theme: %s', (...args) => {
-    expect(getComponentCss(...args)).toMatchSnapshot();
-  });
+  ])(
+    'should return correct css for disabled: %s, hasCustomDropdown: %s, hideLabel: %o, state: %s and theme: %s',
```

Comment:
Shouldn't all boolean values for `hasCustomDropdown` be inverted now since they are effectively causing the opposite now.